README.md

```diff
@@ -13,17 +13,17 @@ Simply give the number of players (with an optional players array), and it will
 var robin = require('roundrobin');
 robin(6);
 [ [ [ 1, 6 ], [ 2, 5 ], [ 3, 4 ] ],
-  [ [ 1, 5 ], [ 6, 4 ], [ 2, 3 ] ],
+  [ [ 5, 1 ], [ 6, 4 ], [ 2, 3 ] ],
   [ [ 1, 4 ], [ 5, 3 ], [ 6, 2 ] ],
-  [ [ 1, 3 ], [ 4, 2 ], [ 5, 6 ] ],
+  [ [ 3, 1 ], [ 4, 2 ], [ 5, 6 ] ],
   [ [ 1, 2 ], [ 3, 6 ], [ 4, 5 ] ] ]
 
 // or with names supplied
 robin(6, ['clux', 'lockjaw', 'pibbz', 'xeno', 'e114', 'eclipse']);
 [ [ [ 'clux', 'eclipse' ], [ 'lockjaw', 'e114' ], [ 'pibbz', 'xeno' ] ],
-  [ [ 'clux', 'e114' ], [ 'eclipse', 'xeno' ], [ 'lockjaw', 'pibbz' ] ],
+  [ [ 'e114', 'clux' ], [ 'eclipse', 'xeno' ], [ 'lockjaw', 'pibbz' ] ],
   [ [ 'clux', 'xeno' ], [ 'e114', 'pibbz' ], [ 'eclipse', 'lockjaw' ] ],
-  [ [ 'clux', 'pibbz' ], [ 'xeno', 'lockjaw' ], [ 'e114', 'eclipse' ] ],
+  [ [ 'pibbz', 'clux', ], [ 'xeno', 'lockjaw' ], [ 'e114', 'eclipse' ] ],
   [ [ 'clux', 'lockjaw' ], [ 'pibbz', 'eclipse' ], [ 'xeno', 'e114' ] ] ]
```

Comment:
There does seem to be an unevenness in the even `n` case (only `clux`, `pibbz`, and `e114`  gets 3 LEFT matches, and the rest getting more RIGHT matches.

There's a benefit to playing more at HOME, and the input list is meant to represent a seeded order.
Perhaps the top n/2 seeds should get the most HOME matches to match this?

Reply:
Seeding was not something I had thought about because usually a round robin is used to seed teams for a bracket style tournament.

If you _did_ want to consider initial seeding going into the round robin as well, the only way that I can think to achieve what you're looking for using the circle method (perhaps you may think of a different way) is to take the initial array of:

`[ 'clux', 'lockjaw', 'pibbz', 'xeno', 'e114', 'eclipse' ]` 

cut it in half into two separate arrays like so:

`[ 'clux', 'lockjaw', 'pibbz' ]` and `[ 'xeno', 'e114', 'eclipse' ]`

and then intermix them back together by alternating pushing the top element of each array into a new array and end up with something like this:

`[ 'xeno', 'clux', 'e114', 'lockjaw', 'eclipse', 'pibbz' ]` 
(perhaps there is a faster/better way to get this output).

You could then take this new array and run it through the same logic I currently have and you would end up with the top 3 teams (`clux` `lockjaw` `pibbz` ) having the most RIGHT hand side (home) matches. This works because the top 3 teams are now occupying the 2nd, 4th and 6th spots.

Note: we would not have to do this for an ODD number of teams as all teams would have the same amount of home games.

Not sure how practical pre-seeding a round robin is though? Let me know what you think!

Reply:
Yeah, I had something similar in mind as well; `[every even indexed player] concat [every odd indexed player]`, and run algorithm as before.

However, the more I think about this, it starts to feel like a bad idea to go to such lengths for something people don't expect. It's probably better to leave this to chance. People can always permute what the player ids represent outside of this.

Reply:
> However, the more I think about this, it starts to feel like a bad idea to go to such lengths for something people don't expect. It's probably better to leave this to chance. People can always permute what the player ids represent outside of this.

I agree! So no pre-seeding then, keep it as is?

Reply:
Yeah, let's keep as is :-)
Resolving.